internal/errors.go

```diff
@@ -18,15 +18,74 @@ package internal
 import (
 	"errors"
 	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"syscall"
 )
 
-var ErrMapKeyMissing = errors.New("map key is required")
-var ErrMapValueMissing = errors.New("map value is required")
-var ErrMapValueAndFileMutuallyExclusive = errors.New("only one of --value and --value-file must be specified")
+// Map errors
+var (
+	ErrMapKeyMissing                    = errors.New("map key is required")
+	ErrMapValueMissing                  = errors.New("map value is required")
+	ErrMapValueAndFileMutuallyExclusive = errors.New("only one of --value and --value-file must be specified")
+)
+
+// Cluster errors
+var (
+	ErrRestAPIDisabled = errors.New("Cannot access Hazelcast REST API. Is it enabled? Check this link to find out more: " + RESTApiDocs)
+)
+
+// Network errors
+var (
+	ErrConnectionTimeout = errors.New("Can not connect to Hazelcast Cluster. Please make sure Hazelcast cluster is reachable, up and running. Check this link to create a IMDG cluster: " + QuickStartDocs)
```

Comment:
Go error messages should start with lower cased letters (that doesn't apply to application error messages)! But I don't think we need to define these errors.